### ISSUE TITLE
add data saving routine

### ISSUE THREAD

**Issue by BradSchoenrock** [BradSchoenrock/WVD-MCSupdate#9]
Need to save data out in NetCDF format. Possibly get matlab routines integrated into labview VIs or work on writing labview VIs to handle NetCDF write output. 

What information needs to be saved into NetCDF output files? 
Where can i get the matlab routines? 
Where can i find documentation on NetCDF formats? 

Brad Schoenrock

**Comment by BradSchoenrock** [comment]
According to Scott Spuler 

> You may be able to find subroutines in Labview.  I know they are available for Matlab so we could use an embedded Matab script

Brad Schoenrock

**Comment by BradSchoenrock** [comment]
Is this included in WVD2 software?

**Comment by BradSchoenrock** [comment]
WVD2 software does save out data files. WVD2 software needs to be refactored but that will be opened in a new issue.

**Comment by BradSchoenrock** [comment]
Data being saved isn't in NETcdf format, it is in a binary format.

**Comment by BradSchoenrock** [comment]
Back to these questions btw... 

What information needs to be saved into NetCDF output files?
Where can i get the matlab routines?
Where can i find documentation on NetCDF formats?

**Comment by BradSchoenrock** [comment]
Ensure Data structure matches GV-HSRL as closely as is reasonable.

**Comment by stillwer** [comment]
As far as I can tell, there is no easy way within Labview to write netcdf files. However, matlab can do it and the matlab script wrapped with labview is, I think, the best plan. I have been testing matlab routines to stream data to a netcdf file and have them working as follows: 1) I can initialize the file, 2) I can write global attributes and define variables, and 3) I can write updated data to files with just the name. Plan to try to write files in parallel with the binary format to start when the software has been ported over to the new computer.

**Comment by stillwer** [comment]
Also...are we trying to match the GV HSRL or are we trying to go with the CF radial format?

**Comment by BradSchoenrock** [comment]
I've heard arguments for both matching the GV HSRL and trying to go with CF radial format. I think it would be good if we could get both the GV HSRL and the WVD onto CF radial in the end. 

I think this enhancement should be built upon the new infrastructure though, so i wouldn't sweat trying to get it going right away once we have the software ported onto the new computer. 

Related isssue #39

**Comment by stillwer** [comment]
I think this is done. The data is being saved. I am currently testing the data, but I think most of the pieces are there.
